src/app/components/inputs/select-input/select-input.component.ts

```diff
@@ -111,12 +107,24 @@ export class SelectInputComponent extends BaseComponent {
         .dropdown(options);
     }
 
-    if (isPlatformBrowser(this.platformId)) {
-      if (this._data && this.control.value) {
-        setTimeout(() => {
-          jQuery(this.select.nativeElement).dropdown("set selected", this.control.value);
-        }, 1);
-      }
+    this.setjQueryOptionValues();
+  }
+
+  private setjQueryOptionValues() {
+    if (!isPlatformBrowser(this.platformId)) return;
+    if (!this._data) return;
+    if (!this.control.value) return;
+
+    let values = this.control.value;
+
+    if (!Array.isArray(values)) {
+      values = [values]
     }
+
+    values.forEach(value => {
+      setTimeout(() => {
+        jQuery(this.select.nativeElement).dropdown("set selected", value);
+      }, 1);
```

Comment:
This code is to handle multi- and single-selects. I don't like checking for an array type and I'll rather just assume an array if its a multi-select, instead of checking for it.